### PR TITLE
improve datafeeder integration

### PR DESCRIPTION
**Pull request by landryb**
now that https://github.com/georchestra/georchestra/pull/3427 and https://github.com/georchestra/datadir/pull/211 have been merged in master, this should allow to simplify the datafeeder integration done in #77

### CODE REVIEW

File: roles/georchestra/tasks/wars.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@
 - name: symlink webapps in each tomcat instance
   file:
     owner: tomcat
-    src: "/usr/share/lib/{{ item.value.pkg }}/{{ item.key }}-generic.war"
+    src: "/usr/share/lib/{{ item.value.pkg }}/{{ item.key }}{% if item.key != 'import' and item.key != 'geowebcache' %}-generic{% endif %}.war"
```
Comment (pmauduit) [inline roles/georchestra/tasks/wars.yml:64]: too bad, it's not harmonized across packages :-/

File: roles/georchestra/templates/datafeeder/datafeeder.service.j2 [hunk 2]
```diff
@@ -1,10 +1,10 @@
 [Unit]
-Description=datafeeder
+Description=geOrchestra datafeeder backend
 After=syslog.target
 
 [Service]
 User=www-data
-ExecStart=/usr/bin/java -Dserver.port={{ datafeeder.port }} -jar /srv/webapps/datafeeder.jar
+ExecStart=/usr/bin/java -Dserver.port={{ datafeeder.port }} -jar /usr/share/lib/georchestra-datafeeder/datafeeder-bin.jar
```
Comment (pmauduit) [inline roles/georchestra/templates/datafeeder/datafeeder.service.j2:7]: I find the jar renaming quite curious, did I miss something in the geor/geor changes about it ?

**Review by pmauduit — APPROVED** [review]

**Comment by landryb** [comment]
this depends on georchestra/georchestra#3487 to produce a proper war for datafeeder-ui and georchestra/georchestra#3488 to give a chance to the buildbot to produce and publish debian packages from the master branch.

File: roles/georchestra/tasks/wars.yml (hunk 1 above)
Comment (landryb) [reply roles/georchestra/tasks/wars.yml]: same thing for geonetwork actually, cf f6b81af

File: roles/georchestra/templates/datafeeder/datafeeder.service.j2 (hunk 2 above)
Comment (landryb) [reply roles/georchestra/templates/datafeeder/datafeeder.service.j2:7]: the build produces a small `datafeeder.jar` and a larger `datafeeder-bin.jar` which iirc is standalone.
```
-rw-r--r-- 1 landry users 91860381 Aug 24 12:16 datafeeder-bin.jar
-rw-r--r-- 1 landry users   890368 Aug 24 12:16 datafeeder.jar
```

from testing (cf georchestra/georchestra@b88c429) i had to use the large one.

**Comment by landryb** [comment]
wrt f6b81af and c1a9924 ive just been confused by my testing of gn4 (which is `geonetwork.war`) while geonetwork3 debian package ships `geonetwork-generic.war`

**Pull request merged**